components/switch/README.md

```diff
@@ -71,13 +71,27 @@ You can also add labels for your checked and unchecked values by using component
 </gameface-switch>
 ```
 
+## Switch attributes
+
 You can use the following attributes to customize the switch
 
 | Attribute           | Type    | Default | Description                                | Notes                                                                                                                                                                                                                 |
 | ------------------- | ------- | ------- | ------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| type              | String  | default       | The type of the switch       | It can be default, inset or text-inside                                                                                                                                                                       |
+| type              | String  | inset       | The type of the switch       | It can be inset or text-inside. If is set something else the type will fallback to inset.                                                                                                                                                                   |
```

Comment:
```suggestion
| type              | String  | | The type of the switch       | It can be without a type, `inset` or `text-inside`. If is set something else the type will fallback to the default.                                                                                                                                                                   |
```

Reply:
Fixed